### ISSUE TITLE
v2.1.19 - Can't edit style info or descriptions

### ISSUE THREAD

**Issue by AdrienM76** [geoffhumphrey/brewcompetitiononlineentry#1192]
Version: v2.1.19

Installation URL: https://bataille.deadyeast.org

Is your installation hosted on brewcompetition.com or brewcomp.com? no

Description of Issue:

This might be a feature rather than a bug, but none of the default BJCP styles can be edited. (the pencil icon is greyed out and inactive)

We wanted to accept 21B Specialty IPAs this year, that don't have official designations. (the guidelines specifically say expansion of this sub-category is possible, and how to indicate a non-defined style) However, we ran into a problem with 2 people trying to submit IPAs with prominent fruit and/or spice additions. (which would mean they belong in 29A, 29B or 30A, which we were not accepting)

I was able to create a Custom IPA category and could specify there that such beers were not allowed, but I could not do so for the general 21B Specialty IPA sub-style as it is uneditable. (which exists precisely for what I was trying to use it for)

While the guidelines don't state in any regular style that such beers don't belong there (they save this for the later category descriptions) it would help if the Required Info & Description texts were editable for 21B. (or otherwise were appended with something like: "*Fruit, Spice, Vegetable, Smoked, or Wood Aged beers are NOT allowed*"

**Comment by geoffhumphrey** [comment]
While not possible via the UI, any style description can be edited if you have direct access to your installation's the styles DB table (via phpMyAdmin, shell access, or the like).

**Comment by AdrienM76** [comment]
Fair enough. Certainly I can do so, but it would be nice not to have them overwritten on an update, thus editable from the UI. I can live with manually maintaining it though.
